shared-libs/transitions/src/transitions/registration.js

```diff
@@ -24,11 +24,11 @@ const findFirstDefinedValue = (doc, fields) => {
   return definedField && doc.fields[definedField];
 };
 
-const getRegistrations = (patientId) => {
-  if (!patientId) {
+const getRegistrations = (subjectId) => {
```

Comment:
Nice attention to detail and renaming these. They'd have worked anyway, but been confusing over time.